sky/data/storage.py

```diff
@@ -3547,3 +3607,450 @@ def _delete_cos_bucket(self):
+                    with ux_utils.print_exception_no_traceback():
+                        raise exceptions.StorageBucketGetError(
+                            'Attempted to connect to a non-existent bucket: '
+                            f'{self.source}') from e
+                else:
+                    # If bucket cannot be found (i.e., does not exist), it is
+                    # to be created by Sky. However, creation is skipped if
+                    # Store object is being reconstructed for deletion.
+                    if self.sync_on_reconstruction:
+                        bucket = self._create_oci_bucket(self.name)
+                        return bucket, True
+                    else:
+                        return None, False
+            elif e.status == 401:  # Unauthorized
+                # AccessDenied error for buckets that are private and not
+                # owned by user.
+                command = (
+                    f'oci os object list --namespace-name {self.namespace} '
+                    f'--bucket-name {self.name}')
+                with ux_utils.print_exception_no_traceback():
+                    raise exceptions.StorageBucketGetError(
+                        _BUCKET_FAIL_TO_CONNECT_MESSAGE.format(name=self.name) +
+                        f' To debug, consider running `{command}`.') from e
+            else:
+                # Unknown / unexpected error happened. This might happen when
+                # Object storage service itself functions not normal (e.g.
+                # maintainance event causes internal server error or request
+                # timeout, etc).
+                with ux_utils.print_exception_no_traceback():
+                    raise exceptions.StorageBucketGetError(
+                        f'Failed to connect to OCI bucket {self.name}') from e
+
+    def mount_command(self, mount_path: str) -> str:
+        """Returns the command to mount the bucket to the mount_path.
+
+        Uses gcsfuse to mount the bucket.
+
+        Args:
+          mount_path: str; Path to mount the bucket to.
+        """
+        # pylint: disable=line-too-long
+        install_cmd = (
+            f'(which dpkg > /dev/null 2>&1 && (which rclone > /dev/null || (cd ~ > /dev/null'
+            f' && curl -O https://downloads.rclone.org/{self.RCLONE_VERSION}/rclone-{self.RCLONE_VERSION}-linux-amd64.deb'
+            f' && sudo dpkg -i rclone-{self.RCLONE_VERSION}-linux-amd64.deb'
+            f' && rm -f rclone-{self.RCLONE_VERSION}-linux-amd64.deb)))'
+            f' || (which rclone > /dev/null || (cd ~ > /dev/null'
+            f' && curl -O https://downloads.rclone.org/{self.RCLONE_VERSION}/rclone-{self.RCLONE_VERSION}-linux-amd64.rpm'
+            f' && sudo yum --nogpgcheck install rclone-{self.RCLONE_VERSION}-linux-amd64.rpm -y'
+            f' && rm -f rclone-{self.RCLONE_VERSION}-linux-amd64.rpm))')
+
+        # pylint: disable=line-too-long
+        mount_cmd = (
```

Comment:
lets put some of the commands in this fucntion to `mounting_utils`. Also, can we reuse some utility functions? Reference:

https://github.com/skypilot-org/skypilot/blob/2bd7c3ed35ce27d5ffeb9010e23da8d9ebb3ffa7/sky/data/storage.py#L1879-L1893

Reply:
Will move the commands to mounting_utils to align with other clouds. For reuse, looks there are not good ones to reuse.